interfaces/builtin/audio_playback.go

```diff
@@ -78,6 +80,12 @@ owner /{,var/}run/user/*/pulse/native rwk,
 owner /{,var/}run/user/*/pulse/pid r,
 `
 
+const audioPlaybackConnectedPlugAppArmorCore = `
+owner /run/user/[0-9]*/###PLUG_SECURITY_TAGS###/pulse/ r,
+owner /run/user/[0-9]*/###PLUG_SECURITY_TAGS###/pulse/native rwk,
+owner /run/user/[0-9]*/###PLUG_SECURITY_TAGS###/pulse/pid r,
```

Comment:
I agree with Alex that it'd be clearer to use SLOT_SECURITY_TAGS as the placeholder here. While this is part of a plug-side rule, we're replacing it with a string derived from the name of the slot snap in the connection.

That's the pattern used in e.g. the `network-manager` interface ConnectedPlugAppArmor rules.

Reply:
Done.